libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -814,6 +816,26 @@ TEST( Full_PKCS11_Capabilities, AFQP_Capabilities )
     xResult = pxGlobalFunctionList->C_GetMechanismInfo( pxSlotId[ 0 ], CKM_SHA256, &MechanismInfo );
     TEST_ASSERT_TRUE( CKR_OK == xResult );
     TEST_ASSERT_TRUE( 0 != ( CKF_DIGEST & MechanismInfo.flags ) );
+
+    /* Check for consistency between static configuration and runtime key
```

Comment:
This is a good idea. 

Could we also do checks on pkcs11testRSA_KEY_SUPPORT and pkcs11testEC_KEY_SUPPORT?